config/index.js

```diff
@@ -1,10 +1,10 @@
 /* eslint no-process-env: "off" */
 require('dotenv').config()
 
-const { buildUrl } = require('../common/helpers/url')
-
 const IS_DEV = process.env.NODE_ENV !== 'production'
 const IS_PRODUCTION = process.env.NODE_ENV === 'production'
+const SERVER_HOST = process.env.SERVER_HOST
+const BASE_URL = `${IS_PRODUCTION ? 'https' : 'http'}://${SERVER_HOST}`
```

Comment:
Could you use a property on the request to do this switch? e.g. https://stackoverflow.com/questions/10348906/how-to-know-if-a-request-is-http-or-https-in-node-js suggests `req.connection.encrypted`

Reply:
Sooo, yes that would be the ideal but at this point the config has no idea about the request.

I've added [an issue]() to see if we can move the setting of the return URL to the controller that makes the request.